docs/robots/ridgeback/maintenance_ridgeback.mdx

```diff
@@ -4,9 +4,117 @@ sidebar_position: 4
 ---
 
 import ComponentIntroductionRidgeback from "/components/introduction_ridgeback.mdx";
+import GettingNewPackages from "/components/maintenance/getting_new_packages.mdx";
+import Support from "/components/support.mdx";
 
 <ComponentIntroductionRidgeback />
 
-:::warning todo—under construction
+## Battery Replacement
+
+If Ridgeback's batteries need replacement, they are accessible by removing the top-plate and the insulator
+cover. Before performing any service or maintenance to the robot, the battery pack must be fully disconnected.
+The batteries weigh approximately 33 kg (73 lbs) each, so use all necessary care and caution when lifting the
+batteries out of the chassis. Please contact Clearpath Robotics regarding replacement batteries.
+
+## Long-term storage
+
+When storing Ridgeback for long periods of time, its important to properly maintain the batteries to fully
+maximize their life. Consider one of the following two procedures when placing Ridgeback in long-term storage:
+
+- Fully charge Ridgeback, turn it off and put it into storage. Once a week, connect power to the charger
+  and allow the charger to top up the battery for an hour or so.
+- Fully charge Ridgeback, turn it off and put it into storage, but leave the charger connected and powered
+  the entire time Ridgeback is in storage. The charger will monitor the battery and will automatically
+  charge it up as needed.
+
+Please contact Clearpath Robotics for additional information about Ridgeback's batteries.
+
+---
+
+## Software Maintenance {#software_maintenance}
+
+<GettingNewPackages />
+
+### MCU Firmware Update
+
+When you update packages, there is periodically a new version of Ridgeback's firmware available. You will know this
+is the case in one of two ways:
+
+1. The firmware and PC are unable to connect to each other, which will be apparent if the body lights remain solid
```

Comment:
change 'PC' to 'Onboard Computer' so it is aligned with the convention in other documentation pages.

Reply:
Updated in 3 spots.